### ISSUE TITLE
bug: undefined variable ref in doc source - ks_kind_port_num

### ISSUE THREAD

**Issue by MikeSpreitzer** [kubestellar/kubestellar#1194]
### Describe the bug

When I look at https://docs.kubestellar.io/main/Coding%20Milestones/PoC2023q1/environments/dev-env/#create-a-kind-cluster-with-a-port-mapping right now, the displayed port mapping section looks like this:

```
  extraPortMappings:
  - containerPort: 443
    hostPort: {{ no such element: mkdocs.config.defaults.MkDocsConfig object['ks_kind_port_num'] }}
    protocol: TCP
```

### Steps To Reproduce

See above

### Expected Behavior

No doc built complaints shown

**Comment by clubanderson** [comment]
ahh, when #1116 merges, this issue will go away.

**Comment by clubanderson** [comment]
fixed now that #1116 is merged
